manifests/package/gentoo.pp

```diff
@@ -1,17 +1,25 @@
-class puppet::package::gentoo {
+class puppet::package::gentoo (
+  $keywords,
+  $use,
+) {
 
-  include puppet::agent
   include puppet::params
 
   if $puppet::server::master {
     include puppet::server
-    $keywords = $puppet::server::gentoo_keywords
     $package  = $puppet::params::master_package
-    $use      = $puppet::server::gentoo_use
+    if $use {
+      $real_use = $use
+    } else {
+      $real_use = '-minimal'
+    }
   } else {
-    $keywords = $puppet::agent::gentoo_keywords
     $package  = $puppet::params::agent_package
-    $use      = $puppet::agent::gentoo_use
+    if $use {
+      $real_use = $use
+    } else {
+      $real_use = 'minimal'
+    }
```

Comment:
why not [selectors?](https://docs.puppetlabs.com/puppet/3.7/reference/lang_conditional.html#selectors)